ops/charm.py

```diff
@@ -392,6 +392,8 @@ class LeaderSettingsChangedEvent(HookEvent):
     This event has been deprecated in favor of using a Peer relation,
     and having the leader set a value in the Application data bag for
     that peer relation.  (see :class:`RelationChangedEvent`).
+
+    .. deprecated:: 2.4.0
```

Comment:
Thanks.

I think we should also get rid of the word "DEPRECATED" at the top, given the docs now say it clearly with the version: https://ops--1178.org.readthedocs.build/en/1178/#ops.LeaderSettingsChangedEvent ... or maybe it's still clearer with both. What do you think, @tonyandrewmeyer?

Reply:
Yeah, I was thinking the same thing - it's cleaner with it removed and the note is visible enough that people should notice it.